x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -948,4 +948,34 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
     ),
     license: 'platinum',
   },
+  {
+    key: 'linux.advanced.fanotify.ignore_unknown_filesystems',
+    first_supported_version: '8.4',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.linux.advanced.fanotify.ignore_unknown_filesystems',
+      {
+        defaultMessage: 'Ignore unknown filesystems.  Default: true',
+      }
+    ),
+  },
+  {
+    key: 'linux.advanced.fanotify.monitored_filesystems',
+    first_supported_version: '8.4',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.linux.advanced.fanotify.monitored_filesystems',
+      {
+        defaultMessage: 'Monitored filesystems.  Default: Endpoint perogative.',
```

Comment:
Is this the text that displayed when you click the question mark next to the option name?

If so, can we add some info about the option being a comma separated list of filesystems? Perhaps an example?